llvm/lib/Target/X86/MCTargetDesc/X86InstPrinterCommon.cpp

```diff
@@ -322,7 +322,7 @@ void X86InstPrinterCommon::printPCRelImm(const MCInst *MI, uint64_t Address,
 
   const MCOperand &Op = MI->getOperand(OpNo);
   if (Op.isImm()) {
-    O << markup("<imm:");
+    O << markup("<addr:");
```

Comment:
Why no `pcrel`? The function is called `printPCRelImm`.

Reply:
Can you guess? Because the name of that function is a lie :sadparrot:.